kibbeh/src/styles/globals.css

```diff
@@ -116,6 +116,19 @@ textarea {
   z-index: 9999999;
 }
 
+#nprogress .bar {
+  background: var(--color-primary-600) !important;
```

Comment:
why this color its barely even visible?

Reply:
https://github.com/benawad/dogehouse/pull/2607#issuecomment-833746758

Reply:
Changed it and I think it looks better now